bears/r/FormatRBear.py

```diff
@@ -14,8 +19,51 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH):
+        """
+        :param r_keep_comments:
+            Determines whether comments are kept or not.
+        :param r_keep_blank_lines:
+            Determines whether blank lines are kept or not.
+        :param r_braces_on_next_line:
+            Determines whether a brace should be placed on the next line.
```

Comment:
I would add an empty line here, then the example gets rendered in a new paragraph :+1: